app/controllers/posts_controller.rb

```diff
@@ -6,6 +6,6 @@ def index
 
   def show
     @post = Post.find(params[:id])
-    @comments = @post.comments
+    @comments = @post.recent_comments
```

Comment:
- [ ] Please make sure to display only `5 recent comments` on the user posts page
<img width="1434" alt="Screenshot 2023-11-17 at 17 15 57" src="https://github.com/Armynerh/my_blog/assets/44745855/d8cea42a-5a12-424c-b483-07f794fd1262">

Reply:
- All comments should be displayed once you are on the post page like this:
<img width="1440" alt="Screenshot 2023-11-17 at 17 17 22" src="https://github.com/Armynerh/my_blog/assets/44745855/11d8d9da-d80c-4aac-825c-317a3641dd36">